lib/coursemology/evaluator/models/programming_evaluation.rb

```diff
@@ -30,7 +30,7 @@ def language=(language)
   def package
     @package ||= begin
       body = plain_request('courses/assessment/programming_evaluations/:id/package', id: id)
-      Package.new(StringIO.new(body))
+      Package.new(Coursemology::Evaluator::StringIO.new(body))
```

Comment:
It doesn't, it's a hack...

And Rubyzip is so nasty I'm not sure I want to PR to fix it there... it's probably because it's a port from Java.